spec/features/users/facebook_login_spec.rb

```diff
@@ -33,9 +33,9 @@
         last_name = "Moidustein"
 
         visit "/"
-        click_on "Register"
+        click_on "Sign Up"
         expect(current_path).to eq("/users/new")
-        click_on "Connect through Facebook"
+        find('#facebook_sign_in').click
 
         expect(current_path).to eq("/profile")
```

Comment:
It looks like this is where the build is failing.  We are ending up on `/users/new` rather than `/profile`. Do you think this is a problem with Capybara not actually clicking the Facebook button?

Reply:
I tried it on local host and the problem appears to be with FB oauth. I've been having trouble with application.yml so I was hoping it would miraculously work on Travis. Sage and I are gonna troubleshoot on call shortly so hopefully will get it passing soon!